test/clients/RestrictedWordApiClient.test.ts

```diff
@@ -0,0 +1,263 @@
+            "./axiosInstance": mockAxiosInstance,
+            "../config": {
+                applicationNamespace: "testNamespace"
+            },
+            "ch-structured-logging": {
+                createLogger: function () {
+                    return mockApplicationLogger;
+                }
+            }
+        });
+
+        return client;
+    };
+
+    const testErrorResponse = {
+        response: {
+            data: {
+                errors: ["Test error"]
+            }
+        }
+    };
+
+    const testUser = "test@user.com";
+    const testWord = "Test word";
+    const testId = "abc123";
+
+    beforeEach(function () {
+
+        sandbox = sinon.createSandbox();
+
+        mockAxiosInstance.post = sinon.stub();
+        mockAxiosInstance.delete = sinon.stub();
+        mockAxiosInstance.get = sinon.stub();
+
+        apiClient = new (requireApiClient())(testUser);
+    });
+
+    afterEach(function () {
+        sandbox.restore();
+    });
+
+    describe("#getAllRestrictedWords", function () {
+
+        const testResults = {
+            data: [
+                {
+                    id: "1",
+                    "full_word": "FIRST",
+                    "created_by": "Fred Jones",
+                    "created_at": "2020-01-23T12:05:08.096"
+                },
+                {
+                    id: "2",
+                    "full_word": "Second",
+                    "created_by": "Jill Jones",
+                    "created_at": "2020-01-24T12:05:08.096",
+                    "deleted_by": "Ben Gun",
+                    "deleted_at": "2020-02-21T11:03:04.019",
+                    deleted: true
+                }
+            ]
+        };
+
+        it("successfully maps results with words", async function () {
+
+            mockAxiosInstance.get.resolves(testResults);
+
+            const options: RestrictedWordQueryOptions = {};
+            const queryString: RestrictedWordFilterDto = {};
+
+            const results = await apiClient.getAllRestrictedWords(options);
+
+            expect(mockAxiosInstance.get).to.have.been.calledWithExactly("/word", {
+                params: queryString
+            });
+
+            const mappedResults = [
+                {
+                    id: "1",
+                    word: "FIRST",
+                    createdBy: "Fred Jones",
+                    deletedBy: undefined,
+                    createdAt: "23 Jan 20",
+                    deletedAt: "-",
+                    deleted: false
+                },
+                {
+                    id: "2",
+                    word: "Second",
+                    createdBy: "Jill Jones",
+                    deletedBy: "Ben Gun",
+                    createdAt: "24 Jan 20",
+                    deletedAt: "21 Feb 20",
+                    deleted: true
+                }
+            ];
+
+            expect(mappedResults).to.eql(results);
```

Comment:
Should use .deep.equal and .equal, not .eql. It's a pretty horrible standard, I wrongly assumed eql was short for equal, not deep equal and probably wont be the only person

Reply:
Further, chai expect is generally plain english. eql is against this idea.